### ISSUE TITLE
Can't add Torrentday

### ISSUE THREAD

**Issue by slinger55** [Jackett/Jackett#581]
Hello,
Over the last 48 hours, TD stopped working completely and refused to search. I deleted it and now when I try and re-add it I get the error - Exception in GetConfigForm so I cannot actually input any information. I am using the latest version of Jackett. Any insight would be great!

**Comment by RageInvader** [comment]
The url that jackett uses is 'tdonline.org' which has now been blocked by most isp's, they have updated the url but just waiting on them building a new release version so I can get it reinstalled onto my server.

**Comment by slinger55** [comment]
Thank you very much for your help. I use Google DNS which was never previously blocked by now is. Cheers.

**Comment by kaso17** [comment]
duplicate of #574
